src/test/java/com/iexec/standalone/result/ResultServiceTests.java

```diff
@@ -128,6 +129,8 @@ class ResultServiceTests {
     @InjectMocks
     private ResultService resultService;
 
+    private ResultService spyResultService;
```

Comment:
2 notes there:
1. Why do you need to create a new instance of `ResultService`? Can't you reuse the other one?
2. You can create a Spy using the `@Spy` annotation.